n_t/Snakefile

```diff
@@ -3,12 +3,21 @@ import pathlib
 import stairway
 import simulations
 import plots
+import numpy as np
+import sys
 
 stairwayplot_code = "stairwayplot/swarmops.jar"
 
+seed = 12345
+np.random.seed(seed)
+
+reps = 2
+seed_array = np.random.random_integers(1,2**32,reps)
```

Comment:
The upper limit for seeds is 2**31 in msprime.

Reply:
i'll make this edit later this morning.

Reply:
fixed